Mayhemfile

```diff
@@ -0,0 +1,6 @@
+project: tiny-lisp
+target: tiny-lisp
+
+cmds:
+  - cmd: /repo/tiny-lisp /test.lisp
```

Comment:
Is the impact here that a malformed lisp file can crash the interpreter? something else? could you possibly add a quick comment above?

Reply:
I think this happened after we updated Mayhemfiles because they were using the `filepath` variable incorrectly. But this is not necessary. Honestly, if the target doesn't check for filename or file extension, you can omit it completely, and the Mayhemfile can look like this:
```
project: tiny-lisp
target: tiny-lisp

cmds:
  - cmd: /repo/tiny-lisp @@
```